src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import { AnimatedCircularProgress } from 'react-native-circular-progress';
+
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../../loc');
+
+interface Props {
+  balance: number;
+  threshold: number;
+}
+
+export const WalletBalanceCircle: FC<Props> = ({ balance = 0, threshold }: Props) => (
```

Comment:
`export const WalletBalanceCircle: FC<Props> = ({ balance = 0, threshold }) => (`
`} :Props` is not needed. 
`balance` is required field so `{ balance, threshold }` should be fine or make balance not required field